openapi/azure.keyvault/Package.md

```diff
@@ -0,0 +1,19 @@
+
+Connects to [Azure Key Vault API](https://azure.microsoft.com/en-us/services/key-vault/) from Ballerina
+
+### Package overview
+
+The `azure.keyvault` is a [Ballerina](https://ballerina.io/) connector for connecting to Azure Key Vault.
+
+#### Compatibility
+|                            | Version               |
+|----------------------------|-----------------------|
+| Ballerina Language         | **Swan Lake Beta2**   |
```

Comment:
**

Reply:
No , mistakenly added it. Thanks for pointing out. Fixed in the PR